spec/active_record/events_spec.rb

```diff
@@ -32,6 +32,14 @@
     expect(task.completed_at).to eq(Time.current)
   end
 
+  it 'On non-persisted object, it updates the timestamp' do
```

Comment:
I think it's a perfect place to use an [RSpec context](https://www.betterspecs.org/#contexts):

```ruby
context 'with a non-persisted object' do
  it 'updates the timestamp' do
    # ...
  end
end
```

We could even do something like this:

```ruby
context 'with a non-persisted object' do
  let!(:task) { build(:task, completed_at: 3.days.ago) }

  it 'updates the timestamp' do
    task.complete!
    expect(task.completed_at).to eq(Time.current)
  end
end
```

Reply:
👍

I moved this into a context, I kept all the setup within the `it` block as it felt more consistent with the other tests (Plus https://thoughtbot.com/blog/lets-not made me think that's the right choice).